### ISSUE TITLE
Another ZeldaGraphGrammar failure

### ISSUE THREAD

**Issue by schrum2** [schrum2/MM-NEAT#391]
Maven build currently reports this:
Tests in error: 
  test(edu.southwestern.tasks.gvgai.zelda.level.ZeldaGraphGrammarTest)

This is an error that I definitely want fixed, but it should be low priority for now. I think that all other issues for prepping the study need to be addressed first, since they will keep changing the code and therefore the test behavior anyway. Only after we've stopped adding features should you settle down one last time to fix these tests.

Still, taking a quick look at the error to see if there is in fact a problem could be helpful.

**Comment by jakeryang** [comment]
I'm not getting any build errors when running the test on its own or when running a maven install.

**Comment by schrum2** [comment]
Hmmm ... I wonder if there is some uncommitted file I don't have.

In any case, just ignore this issue for now. I'll try to give you more information, but if it keeps not being a problem I'll eventually close it.

**Comment by schrum2** [comment]
@jakeryang This is still an issue for me ... I've disabled the test. It happens during Maven build.

**Comment by schrum2** [comment]
Ignore
